src/checks/ball.js

```diff
@@ -0,0 +1,34 @@
+const _ = require('lodash');
+const ball_data = require('../helpers/ball-data.js');
```

Comment:
Can we use camelCase for these variable names? (Yes, I know I was the one who originally wrote this code.)